src/WebJobs.Extensions.DurableTask/Listener/TaskEntityShim.cs

```diff
@@ -148,6 +148,7 @@ internal void Rehydrate(string serializedInput)
 
         public override async Task<string> Execute(OrchestrationContext innerContext, string serializedInput)
         {
+            string status;
```

Comment:
Consider naming this variable `statusForTracking`, or something similar to help communicate the purpose of this data.

Reply:
Did so and left an explanatory comment as well :)